<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveRecordDialog</class>
 <widget class="QDialog" name="SaveRecordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tetris</string>
  </property>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please, enter your name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="gameOver">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>GAME OVER</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Score: </string>
   </property>
  </widget>
  <widget class="QLabel" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Line: </string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreResult">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>244</string>
   </property>
  </widget>
  <widget class="QLabel" name="lineResult">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
